### PR TITLE
MOP-123 Refactor to fetch user details from manage-users-api rather than auth

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: server/services/userService.ts [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
 export default class UserService {
-  constructor(private readonly hmppsAuthClient: HmppsAuthClient) {}
+  constructor(private readonly manageUsersApiClient: ManageUsersApiClient) {}
 
-  async getUser(userDetails: User): Promise<UserDetails> {
-    const user = await this.hmppsAuthClient.getUser(userDetails)
-    return { ...user, displayName: convertToTitleCase(user.name as string) }
+  async getUser(userDetails: Express.User): Promise<UserDetails> {
+    const user = await this.manageUsersApiClient.getUser(userDetails)
+    return { name: user.name, displayName: convertToTitleCase(user.name as string) }
```
Comment (carloveo-moj) [inline server/services/userService.ts:14]:
```suggestion
    return { ...user, name: user.name, displayName: convertToTitleCase(user.name as string) }
```

Not sure if you need to destructure the user here as well?

**Review by carloveo-moj — DISMISSED** [review]
LGTM

File: server/services/userService.ts (hunk 1 above)
Comment (yasinmustafa) [reply server/services/userService.ts]: good spot, the service actually only uses the diaplayName from this object.. but makes sense to destructure here so other attributes are also available if needed in the future.

Fixed and pushed

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Pull request merged**